handlers/feed.go

```diff
@@ -29,10 +33,78 @@ func GetGenericFeed(w http.ResponseWriter, r *http.Request) {
 
 	feed.Value = feeds.AddedValue(feed.Value, tribe.OwnerPubKey)
 
+	processYoutubeDownload(url, *feed)
+
 	w.WriteHeader(http.StatusOK)
 	json.NewEncoder(w).Encode(feed)
 }
 
+func processYoutubeDownload(url string, feed feeds.Feed) {
+	if strings.Contains(url, "youtube") {
+		var data []string
+		for z := 0; z < len(feed.Items); z++ {
+			i := feed.Items[z]
+			data = append(data, i.Link)
+		}
+
+		stakworkKey := fmt.Sprintf("Token token=%s", os.Getenv("STAKWORK_KEY"))
```

Comment:
We should document this new ENV var somewhere, is there a wiki for this repo or in the readme we can add that

Reply:
We shouldn't do this function call either if this key doesn't exist as the request will always fail so we should check for that aswell

Reply:
Ok, thanks @kevkevinpal I have pushed the updates.